src/rules/order.js

```diff
@@ -638,6 +721,29 @@ module.exports = {
               'never',
             ],
           },
+          named: {
+            default: false,
+            oneOf: [{
+              type: 'boolean',
+            }, {
+              type: 'object',
+              oneOf: [{
+                properties: {
+                  enabled: { type: 'boolean' },
+                  groups: groupsType,
+                },
+                additionalProperties: false,
+              }, {
+                properties: {
+                  import: { type: 'boolean' },
+                  export: { type: 'boolean' },
+                  require: { type: 'boolean' },
```

Comment:
i think i'd expect here that `enabled` could always be present, and `import`, `export`, and `require` would just override whatever base `enabled` implies?

Reply:
Sounds like a great idea - I'm not sure how to efficiently aggregate the configuration, then, though
A bunch of `"{key}" in options.named` calls?

Reply:
hm, yeah that'll be tricky. maybe indeed something like:
```js
{
  import: 'import' in options ? options.import : options.enabled,
  export: 'export' in options ? options.export : options.enabled,
  require: 'require' in options ? options.require : options.enabled,
}